### DISCUSSION TITLE
Prisma Throwing a Typeerror After Deployment to Vercel, Problem Narrowed to Extending Prisma Generated Types

### DISCUSSION THREAD

**Discussion by Mnemonic775** [prisma/prisma#21519] [Q&A]
### Bug description

After deploying my app to Vercel I get a 500 error when navigating to some pages with this output:
`
    TypeError: The specifier “.prisma/client/index-browser” was a bare specifier, but was not remapped to anything. Relative module specifiers must start with “./”, “../” or “/”`

I've narrowed the problem down to using these lines to create types using subsets of properties of the generated types that Prisma created based on my schema.

```ts
const studentFull = Prisma.validator<Prisma.StudentDefaultArgs>() ({
    include: { 
        openPeriods:true,
        sessionRecords:true,
        assignedDays:true,
    }
})

type StudentFull = Prisma.StudentGetPayload<typeof studentFull>

```
When you comment out this code and get rid of the typing based on it the page works beautifully, but obviously Typescript argues with me about the typing.

Is there any way to make this work with correct typing? I'd like to get it done correctly using TypeScript.

### How to reproduce

Use Prisma's type extension features then deploy to Vercel.

### Expected behavior

The typing was expected to function as it did when deployed locally.

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider  = "postgresql"
  url       = env("DATABASE_URL")
  directUrl = env("DIRECT_URL")
}

model Student {
  studentName    String                @unique
  studentGrade   String
  openPeriods    StudentAvailability[]
  sessionRecords CompletedSession[]
  assignedDays   TutoringAssignment[]
}

model StudentAvailability {
  id               Int     @id @unique @default(autoincrement())
  student          Student @relation(fields: [studentName], references: [studentName])
  studentName      String
  availablePeriods Boolean[] //0-8 teaching periods
}

model User {
  userName     String  @unique
  userPassword String
  userRole     String
  linkedTutor  Tutor?  @relation(fields: [tutorName], references: [tutorName])
  tutorName    String? @unique
}

model Tutor {
  tutorName         String               @unique
  tutorAvailability DailyAvailibility[]
  tutorRate         Int
  assignedStudents  TutoringAssignment[]
  linkedUser        User?
  completedSessions CompletedSession[]
  tutorPhone        BigInt
  tutorEmail        String
}

model DailyAvailibility {
  id                    Int    @id @unique @default(autoincrement())
  tutor                 Tutor  @relation(fields: [tutorName], references: [tutorName])
  tutorName             String
  dayOfWeek             Int //0 being Sunday and 6 Saturday
  startHours             Int
  startMinutes           Int
  endHours               Int
  endMinutes             Int
}

model TutoringAssignment {
  id             Int     @id @unique @default(autoincrement())
  tutor          Tutor   @relation(fields: [tutorName], references: [tutorName])
  student        Student @relation(fields: [studentName], references: [studentName])
  studentName    String
  tutorName      String
  dayAssigned    Int //0 being Sunday and 6 Saturday
  periodAssigned Int //0-6
}

model CompletedSession {
  id            Int     @id @unique @default(autoincrement())
  tutor         Tutor   @relation(fields: [tutorName], references: [tutorName])
  student       Student @relation(fields: [studentName], references: [studentName])
  studentName   String
  tutorName     String
  sessionRecord String
  sessionMonth  Int
  sessionYear   Int
  sessionDay    Int
  studentPresent Boolean
}

```ts
// Add your code using Prisma Client
```

### Environment & setup

- Database: PostgreSQL
-  Vercel Deployment

### Prisma Version

```
5.4.2

```

**Comment by ludralph** [comment]
Hi @Mnemonic775 👋 

Can you please try the solution in this [issue](https://github.com/prisma/prisma/issues/12504#issuecomment-1285883083) comment?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
